src/scene/materials/standard-material.js

```diff
@@ -727,6 +727,22 @@ class StandardMaterial extends Material {
 
         this.dirtyShader = false;
     }
+
+    /**
+     * @function
+     * @name StandardMaterial#destroy
+     * @description Removes this material from the scene and possibly frees up memory from its shaders (if there are no other materials using it).
+     */
+    destroy() {
+        // unbind (texture) asset references
+        for (const asset in this._assetReferences) {
+            this._assetReferences[asset]._unbind();
+        }
+        this._assetReferences = null;
```

Comment:
Wondering it is safer to make _assetReferences into the state that was in in the constructor (ie `_assetReferences = {}`) like we do in the Material.destroy function

Reply:
But if you call `destroy`, the object is done. Kaput. Finished. Are you expecting it to be used again? I mean, also, that would allocate an object for no decent reason.

Reply:
reverted - if there is expects a valid object it throw an exception - which is good so we can fix that code (hopefully shouldn't be)